CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+## Dev env setup
+
+1. Clone the project
+2. Create a virtualenv
+3. pip install -e .[test]
+
+## Launch tests
+
+With pytest:
+```bash
+pytest tests
```

Comment:
Here are some detailed instructions.
```
# cleanup temp dir
rm -rf /tmp/rd
cd /tmp


# use pyenv to delete virtualenv
#  this assumes you have used pyenv to install a 3.6  (change accordingly to the versions you install)
pyenv virtualenv-delete -f rd
pyenv virtualenv 3.6.11 rd
pyenv local rd

# get the code
git clone https://github.com/GigiusB/django-relativedelta.git .

# swith to branch
git checkout features/tox-introduction

# install project in editable mode (this will install needed dependencies
pip install -e .[test]

# add other environments for tox
# this assumes you have used pyenv to install a 3.7 and 3.8 (change accordingly to the versions you install)
pyenv local rd 3.7.8 3.8.2

# run tox
tox
```

Reply:
OK I got it to work with pytest now. With tox I'm still getting errors about connecting to the database, but those should be resolved if you remove the `PGHOST` stuff

Reply:
Also how is one supposed to override env vars with `tox`? I tried the typical `$ PG_HOST=/var/run/postgresql PG_USER=peter tox`, but that doesn't seem to be honored. With pytest this worked.

Reply:
I suppose we could add some entries like the following to the tox.ini vi pg_probut I'm not overly fond of it:
```pg: PG_PORT = {env:PG_PORT:5432}```
More on this: https://tox.readthedocs.io/en/latest/config.html#environment-variable-substitutions-with-default-values

Instead than cluttering the tox.ini I normally use the very useful [direnv](https://direnv.net/) : it will automatically execute some commands from a .envrc (and there you can use export xyz=xyz...) when you enter a folder (say your project root) and unset them when you cd out of it.

I also prefer to add all my variables in a .env file (no export needed) which is loaded by .envrc filewith the first line "dotenv".

See my .env.example and .envrc.example files